stories/Timeline/Timeline.stories.js

```diff
@@ -0,0 +1,188 @@
+                </CardTitle>
+                <CardText>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                  do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                </CardText>
+                <CardSignature>di Federico De Paolis</CardSignature>
+                <CardReadMore text="Leggi di più" iconName="it-arrow-right" />
+              </CardBody>
+            </Card>
+            {/* end card */}
+          </TimelinePin>
+        </div>
+        <div className="col-12">
+          <TimelinePin label="ottobre 2018">
+            {/* start card */}
+            <Card>
+              <CardBody>
+                <CardTitle>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                  do eiusmod tempor…
+                </CardTitle>
+                <CardText>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                  do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                </CardText>
+              </CardBody>
+            </Card>
+            {/* end card */}
+          </TimelinePin>
+        </div>
+        <div className="col-12">
+          <TimelinePin label="novembre 2018">
+            {/* start card */}
+            <Card>
+              <CardBody>
+                <CardCategory date="10/12/2018">Category</CardCategory>
+                <CardTitle tag="h5" className="big-heading">
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit…
+                </CardTitle>
+                <CardText>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                  do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                </CardText>
+                <CardSignature>di Federico De Paolis</CardSignature>
+                <CardReadMore text="Leggi di più" iconName="it-arrow-right" />
+              </CardBody>
+            </Card>
+            {/* end card */}
+          </TimelinePin>
+        </div>
+        <div className="col-12">
+          <TimelinePin label="dicembre 2018">
+            {/* start card */}
+            <Card>
+              <CardBody>
+                <CardTitle>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                  do eiusmod tempor…
+                </CardTitle>
+                <CardText>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                  do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                </CardText>
+              </CardBody>
+            </Card>
+            {/* end card */}
+          </TimelinePin>
+        </div>
+      </div>
+    </TimelineWrapper>
+  </div>
+)
+
+storiesOf('Componenti/Timeline', module)
+  .addDecorator(withA11y)
+  .add(
+    'Esempi',
+    withInfo({
+      text: Esempi
```

Comment:
Exclude from the info panel all the Cards component:
```js
propTables: [Timeline, TimelinePin],
propTablesExclude: [Card, CardBody, ....]
```

Reply:
Done